src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int topBasis;
+    private int bottomBasis;
+    private int height;
+
+    public IsoscelesTrapezoid(String color, int topBasis, int bottomBasis, int height) {
+        super(color);
+        this.topBasis = topBasis;
+        this.bottomBasis = bottomBasis;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.round(0.5 * (topBasis + bottomBasis) * height);

```

Comment:
why did you round this?
```suggestion
        return 0.5 * (topBasis + bottomBasis) * height;
```

Reply:
I thought that multiplying integers would return an integer. I was wrong